src/components/BulkEnrollmentResultsDownloadPage/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext, useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Redirect, useParams } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { logError } from '@edx/frontend-platform/logging';
+import { camelCaseObject } from '@edx/frontend-platform';
+import { ToastsContext } from '../Toasts';
+import LicenseManagerApiService from '../../data/services/LicenseManagerAPIService';
+
+const BulkEnrollmentResultsDownloadPage = ({ enterpriseId }) => {
+  const { enterpriseSlug, bulkEnrollmentJobId } = useParams();
+  const { addToast } = useContext(ToastsContext);
+  const [isLoading, setLoading] = useState(true);
+  const [redirectUrl, setRedirectUrl] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (enterpriseId && isLoading) {
+      LicenseManagerApiService.fetchBulkEnrollmentJob(enterpriseId, bulkEnrollmentJobId)
+        .then((response) => {
+          const result = camelCaseObject(response.data);
+          if (result.downloadUrl) {
+            setRedirectUrl(result.downloadUrl);
+          } else {
+            throw new Error('Your download is not ready yet.');
+          }
+        })
+        .catch((err) => {
+          logError(err);
```

Comment:
[clarification/confirmation] If the error on line 25 above is thrown any time a user lands on this page (or refreshes) and their download isn't ready, we do in fact want to report that to New Relic via `logError` and count against any alerts that get routed to on-call?

I'm just imagining someone gets the URL to this page and hammers the refresh button (🔨 😅) until they get the download completed, which could be a big burst of errors in a short period of time. If we want to potentially get alerted on bursts like this, `logError` makes sense with the custom "download isn't ready" error! Always something to come back to if it does become noisy and impacting alerts.

Reply:
Thanks. I didn't realize `logError` sent to NewRelic